pagopa/io-app
# chore(Federated Identity Management System): [IAFIMS-7] Handles the remote FIMS domain configuration

## Short description
This PR implements the handling of the remote domain to properly store the FIMS cookie 

## How to test
1. Enables FIMS local FF
2. execute from anywhere:
```typescript
        handleInternalLink(
          dispatch,
          `ioit://${FIMS_ROUTES.WEBVIEW}?url=http://127.0.0.1:3000/myportal_playground.html`
        );
```
check the cookie is correctly passed out and stored to the remotely configured domain.

This PR depends on pagopa/io-dev-api-server#158 pagopa/io-services-metadata#602 https://github.com/pagopa/io-app/pull/3850
